src/Performance/PerformanceConnector.ts

```diff
@@ -57,7 +57,7 @@ export interface PerformanceConverter extends MorningstarConverter {
  * */
 
 export const DATA_TABLES = [
-    { key: 'TestConverter' }
+    { key: 'CalendarYearReturns' }
```

Comment:
Should we just use `CalendarYearReturn` (singular) instead of plural form (`CalendarYearReturns`)? To match the response.